docs/troubleshooting/faqs/General.mdx

```diff
@@ -140,6 +140,51 @@ See [this issue comment](https://github.com/typescript-eslint/typescript-eslint/
 See [Changes to one file are not reflected in linting other files in my IDE](#changes-to-one-file-are-not-reflected-when-linting-other-files-in-my-ide).
 Rules such as [`no-unsafe-argument`](/rules/no-unsafe-argument), [`no-unsafe-assignment`](/rules/no-unsafe-assignment), and [`no-unsafe-call`](/rules/no-unsafe-call) are often impacted.
 
+## How does typescript-eslint compare to native-speed linters?
```

Comment:
Todo for me: run this by native-speed linter folks to make sure this phrasing isn't wrong or accidentally obtuse. We want to be good & friendly partners!

Reply:
Got a private 👍